src/deployment/flavors.md

```diff
@@ -153,18 +153,24 @@ project 'Runner', {
 Setting up flavors in Android can be done in your project's 
 **build.gradle** file.
 
-1. In VSCode, inside your Flutter project, 
-navigate to **android**/**app**/**build.gradle**.     
+1. Inside your Flutter project, 
+   navigate to **android**/**app**/**build.gradle**.
 
-2. Add a flavor object with the specified environments along with values for 
-**dimension**, **resValue**, and **applicationId** or **applicationSuffix**. 
-  * The name of the application for each build is located in **resValue** 
-(as shown in the snippet below from VSCode).
```

Comment:
**question:** Did you remove the screenshot referenced here? If not, is it used elsewhere?

Reply:
This "snippet" is just the code snippet I modified below. The VSCode mention didn't add anything, as they could access/edit the file in any tool.

Reply:
Thanks for the clarification, @parlough.